lineman/app/components/thread_preview_collection/thread_preview_collection.scss

```diff
@@ -18,6 +18,7 @@
 .thread-preview__thread-header {
   @include fontCrapName;
   padding-bottom: 4px;
+  display: flex;
```

Comment:
Could you remind me what this does @HSalmon? I know flexbox rocks but don't quite recall the specifics of how it works.